### PR TITLE
[FIX/EPMEDU-1622] Show feeding point image in details view

### PR DESCRIPTION
**Pull request by ozkarram**
Image data added in details view. Additionally adds image at favorites and search section.

Mock data updated as well.

Update: Uses StorageApi to fetch urls from Amplify.

### CODE REVIEW

File: shared/feature/feeding/src/main/java/com/epmedu/animeal/feeding/data/mapper/OnUpdateFeedingPointToDomainMapper.kt [hunk 1]
```diff
@@ -23,5 +23,8 @@ internal fun OnUpdateFeedingPoint.toDomainFeedingPoint() =
         location = MapLocation(
             latitude = location().lat(),
             longitude = location().lon()
-        )
+        ),
+        images = with(images()) {
```
Comment (TrubkaU) [inline shared/feature/feeding/src/main/java/com/epmedu/animeal/feeding/data/mapper/OnUpdateFeedingPointToDomainMapper.kt]: Maybe it simplyfied?
```suggestion
        images() ?: emptyList()
```

**Review by TrubkaU — APPROVED** [review]

File: shared/feature/feeding/src/main/java/com/epmedu/animeal/feeding/data/mapper/OnUpdateFeedingPointToDomainMapper.kt (hunk 1 above)
Comment (ozkarram) [reply shared/feature/feeding/src/main/java/com/epmedu/animeal/feeding/data/mapper/OnUpdateFeedingPointToDomainMapper.kt]: Looks better, thanks!

File: shared/feature/feeding/src/main/java/com/epmedu/animeal/feeding/presentation/model/FeedingPointModel.kt [hunk 2]
```diff
@@ -19,7 +19,8 @@ data class FeedingPointModel(
     val isFavourite: Boolean = false,
     val lastFeeder: Feeder,
     val remoteness: Remoteness = Remoteness.ANY,
-    val coordinates: Point
+    val coordinates: Point,
+    val images: List<String> = emptyList()
```
Comment (Fileek) [inline shared/feature/feeding/src/main/java/com/epmedu/animeal/feeding/presentation/model/FeedingPointModel.kt]: Let's keep this model simple and store not a list of images, but only a first image. 
You pass only the first one anyway

**Review by Fileek — CHANGES_REQUESTED** [review]

File: shared/feature/feeding/src/main/java/com/epmedu/animeal/feeding/presentation/model/FeedingPointModel.kt (hunk 2 above)
Comment (ozkarram) [reply shared/feature/feeding/src/main/java/com/epmedu/animeal/feeding/presentation/model/FeedingPointModel.kt]: this model only? I was thinking the whole implementation grab first element only and I'm not sure if there is a scenario where we use the whole images list.

**Review by Fileek — APPROVED** [review]

**Pull request merged**